R/cnd-abort.R

```diff
@@ -554,3 +915,26 @@ last_trace <- function() {
 # This is where we save errors for `last_error()`
 last_error_env <- new.env(parent = emptyenv())
 last_error_env$cnd <- NULL
+
+
+#' Development notes - `cnd-abort.R`
```

Comment:
I think this might be a bit too distant to the code it affects and it'd be easy to miss when updating the code. Maybe include in the docs for `error_call()`, but use `#` instead of `#'` so it doesn't become user facing?  Or maybe just inline as comments in the appropriate bit of the code?